src/main/java/Duke.java

```diff
@@ -1,10 +1,30 @@
+import java.util.Scanner;
+
+import Command.Command;
+import Parser.Parser;
+import Tasks.*;
+import Ui.Ui;
+import Storage.TaskFileHandler;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        runDuke();
+    }
+
+    public static void runDuke() {
+        Scanner scanner = new Scanner(System.in);
+        Ui.helloWorld();
+        boolean converse = true;
```

Comment:
boolean should sound like booleans, maybe isConverse is better?

Reply:
good point